internal/handlers/tigris/msg_insert.go

```diff
@@ -107,6 +107,12 @@ func insertMany(ctx context.Context, dbPool *tigrisdb.TigrisDB, fp *tigrisdb.Fet
 	var inserted int32
 	var insErrors common.WriteErrors
 
+	// Attempt to insert all the documents in the same request to make insert faster.
+	if err := dbPool.InsertManyDocuments(ctx, fp.DB, fp.Collection, docs); err == nil {
+		return int32(docs.Len()), &insErrors
```

Comment:
```suggestion
		return int32(docs.Len()), nil
```

The `err` is nil here.

Reply:
It will not work here because the caller expects `insErrors` to be not `nil`. I decided to keep it as is and not refactor it.

Reply:
That was unexpected but I see the logic now :hugs: